wdio.conf.js

```diff
@@ -86,7 +86,7 @@ exports.config = {
         browserName: 'chrome',
         'goog:chromeOptions': {
           args: [
-            'headless',
+            //'headless',
```

Comment:
This seems to work on the headless version, so I'll remove this comment for you.